ports/geckolib/glue.rs

```diff
@@ -2908,7 +2903,7 @@ fn create_context<'a>(per_doc_data: &'a PerDocumentStyleDataImpl,
 
 #[no_mangle]
 pub extern "C" fn Servo_GetComputedKeyframeValues(keyframes: RawGeckoKeyframeListBorrowed,
-                                                  element: RawGeckoElementBorrowed,
+                                                  _element: RawGeckoElementBorrowed,
```

Comment:
Remove the argument (maybe in a followup given this means you need to touch C++ too)?